src/README.md

```diff
@@ -71,7 +71,10 @@ _The file's (module's) content is imported automatically"_
   
 The second step is to determine what events your Hunter will subscribe to, and from where you can get them.  
 `Convention:` Events should be declared in their corresponding module. for example, a KubeDashboardEvent event is declared in the dashboard discovery module.  
-     
+
```

Comment:
This change looks familiar from another PR, so I'd remove it from this one

Reply:
Do you mean that one?
"
`Note:` An hunter located under the `disovery` folder should not import any modules located under the `hunting` folder 
in order to prevent circular dependency bugs 
"